rastervision_core/rastervision/core/data/label/semantic_segmentation_labels.py

```diff
@@ -200,12 +202,22 @@ def from_predictions(cls,
             extent (Box): The extent of the region to which the labels belong,
                 in global coordinates.
             num_classes (int): Number of classes.
+            crop_sz (Optional[int]): Number of rows/columns of pixels from the
```

Comment:
If `crop_sz` is > 0, then it seems like the first window should start at `(-crop_sz, -crop_sz)` instead of `(0, 0)`.

Reply:
Yes, but this is a little tricky. An elegant solution would make use of the existing `padding` param in `Box.get_windows()`. This would require modifying `Box.get_windows()` so that it applies padding to all sides instead of just the right and bottom as it currently does. Let me see if I can do that without breaking anything.

Reply:
The current solution does not change anything in the training defaults. It just switches to symmetrical padding if `crop_sz` is not `None`. The user can choose to retrain using a different padding style if they feel unsatisfied with the predictions.

Reply:
+1